Question/server/middleware.js

```diff
@@ -0,0 +1,53 @@
+const authApi = require('./api/AuthApi.js');
+
+const getToken = async (req, res) => {
+  try {
+    const authApiRes = await authApi.getToken(req.userInfo);
+    res.json({
+      message: 'Generated JWT successfully',
+      token: authApiRes.data.token,
+    });
+  } catch (err) {
+    res
+      .status(err?.response?.status || 500)
+      .json({ error: err?.response?.data?.error || err });
+  }
+};
```

Comment:
Question service does not need to getToken, only for User service to log in

Reply:
removed